src/createNumbersGenerator.js

```diff
@@ -26,7 +26,20 @@
  * @return {number}
  */
 function createNumbersGenerator(numbers, min, max) {
```

Comment:
you can write default values right here inside parentheses

Reply:
> you can write default values right here inside parentheses

refactored